examples/observability/eks-java-jmx/data.tf

```diff
@@ -0,0 +1,11 @@
+data "aws_region" "current" {}
+
+data "aws_availability_zones" "available" {}
+
+data "aws_eks_cluster" "cluster" {
+  name = module.aws-eks-accelerator-for-terraform.eks_cluster_id
+}
+
+data "aws_eks_cluster_auth" "cluster" {
+  name = module.aws-eks-accelerator-for-terraform.eks_cluster_id
+}
```

Comment:
these can go into `main.tf` file. We would like to keep one file for examples to keep ti simple